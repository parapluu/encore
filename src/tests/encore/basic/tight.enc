class Main
  def main(args : [String]) : void
    val a = [1,2,3]
    val b = [4,5]
    if |a|+|b|<|a|!=|b|<=|a| then
      println("You shouldn't be writing code like this...")
    end
  end
end