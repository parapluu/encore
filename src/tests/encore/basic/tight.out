You shouldn't be writing code like this...
